tests/unit/drivers/rank/test_matches2doc_rank_drivers.py

```diff
@@ -40,20 +38,19 @@ def create_document_to_score():
     # |- matches: (id: 3, parent_id: 1, score.value: 3),
     # |- matches: (id: 4, parent_id: 1, score.value: 4),
     # |- matches: (id: 5, parent_id: 1, score.value: 5),
-    doc = Document()
+    doc = jina_pb2.DocumentProto()
```

Comment:
What's the reason? Then which should we use?
Document() is not correct here. Because this line won't set the value. 
https://github.com/jina-ai/jina/blob/f0b6a44045f7fccca05a34020fd42981ce34dc4e/tests/unit/drivers/rank/test_matches2doc_rank_drivers.py#L61

Reply:
Ok I know what to do

Reply:
Then we need to change other jina_pb2.DocumentProto() to Document() in other tests right?